### ISSUE TITLE
Epic: Migrate React Admin

### ISSUE THREAD

**Issue by ObliviousHarmony** [woothemes/woocommerce#32433]
In order to complete the WooCommerce Admin merge, we need to fold the React Admin directly into the Core plugin's source directory. There are a few steps in this process:

- [x] https://github.com/woocommerce/woocommerce/issues/33947
- [x] https://github.com/woocommerce/woocommerce/issues/34918
- [x] https://github.com/woocommerce/woocommerce/issues/33948
- [x] https://github.com/woocommerce/woocommerce/issues/33949

**Comment by jonathansadowski** [comment]
For now, I'm moving this to blocked, as I wasn't able to unblock the in-progress PR in the timebox we agreed to.

**Comment by lsinger** [comment]
Is this still a priority?

**Comment by ObliviousHarmony** [comment]
Closing this in favor of new issues as an output from a larger project to complete the merge.
